googleanalytics/googleAnalyticsReportTool.py

```diff
@@ -0,0 +1,109 @@
+import os
+from superagi.tools.base_tool import BaseTool
+from pydantic import BaseModel, Field
+import json
+from typing import Type, Optional
+from superagi.resource_manager.file_manager import FileManager
+import yaml
+from google.analytics.data_v1beta import BetaAnalyticsDataClient
+from google.analytics.data_v1beta.types import (
+    DateRange,
+    Dimension,
+    Metric,
+    RunReportRequest,
+)
+
+class GoogleAnalyticsReportToolInput(BaseModel):
+    start: str = Field(..., description="The starting date of the query, in YYYY-MM-DD format")
```

Comment:
start->start_date
end->end_date
use appropriate variable name for is_single_file as discussed.